src/components/ButtonChoices.jsx

```diff
@@ -1,18 +1,35 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import { useHistory } from 'react-router-dom';
+import ScoreContext from '../contexts/ScoreContext';
```

Comment:
Imports : 
Dependencies 
Space
Components or Contexts etc
Space
Styles and Assets
(not only for this file)